load-tests/in_memory_hub/deployment/service.yaml

```diff
@@ -0,0 +1,16 @@
+---
+apiVersion: "v1"
+kind: "Service"
+metadata:
+  labels:
+    app: "in_memory_hub"
+  name: "in-memory-hub-svc"
+spec:
+  ports:
+    - name: "port-1-in-mem"
+      port: 9090
+      protocol: "TCP"
+      targetPort: 9090
+  selector:
+    app: "in_memory_hub"
+  type: "ClusterIP"
```

Comment:
Lets add this to end of this file as well. Service for the jmeter pod.
---
```
apiVersion: "v1"
kind: "Service"
metadata:
  labels:
    app: "jmeter"
  name: "client-svc"
spec:
  ports:
    - name: "client-port"
      port: 9100
      protocol: "TCP"
      targetPort: 9100
  selector:
    app: "jmeter"
  type: "ClusterIP"
  ```

Reply:
Fixed with [789a2d2b988146e2d1d9c523c2ad267b0606a535](https://github.com/ballerina-platform/module-ballerina-websubhub/pull/432/commits/789a2d2b988146e2d1d9c523c2ad267b0606a535)